.eslintrc.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
                     },
                     {
                         selector: ['parameter', 'method'],
-                        format: ['camelCase'],
+                        format: ['camelCase', 'PascalCase'],
```

Comment:
Why are we using pascal case for method parameters here?

Reply:
@akinwale It's left from TS migration. It shouldn't hurt, though, cause it's necessary for other HOCs anyway where we have `WrappedComponent` as a param (and it's in CamelCase). But I can remove it from this PR if necessary.